src/questions/data_types.yaml

```diff
@@ -1,37 +1,1106 @@
-  uuid: 1b8fbe3c-aeb2-4b7d-a3ef-3082bf07efcc
-  # Types can be MultipleChoice or SingleChoice
-  type: MultipleChoice
-  # Content should be in Markdown format
-  content: What does SW DAO offer?
-  hint: This is shown if user needs some help
-  explanation: This is shown when the user answers the choice incorrectly
-  answerKeys:
-    - e0kl
-  # should be from one of the predefined list of topics
-  topic: "01_data_types"
-  # should be from one of the predefined list of sub-topics, or can be a new one
-  sub_topics:
-    - "string"
-    - "variables"
-  # Difficulty level should be one option from "Low" , "Medium", "High"
-  difficulty_level: "Low"
-  choices:
-    # Content should be in Markdown format
-    - content: It offers crypto based mining applications for effective utilization
-        of resources.
-      # four character random string
-      key: rtte
-      # order in which the choice should appear
-      order: 0
-    - content: It offers strategic DeFi investment products which allocate capital
-        based on Machine Learning
-      key: e0kl
-      order: 1
-    - content: It offers coding and machine learning solutions to develop Defi
-        applications.
-      key: fklo
-      order: 2
-    - content: All of the about options
-      key: pwlp
-      order: 3
+- uuid: 89128b13-bd0f-4887-85c4-1308dfbc3234
+  type: MultipleChoice
+  content: Which of the following are not valid data types in solidity?
+  hint: noHint
+  explanation: floating point data and character data types are not valid in solidity.
+  answerKeys:
+    - B
+    - D
+  sub_topics:
+    - "value-types"
+  difficulty_level: Low
+  choices:
+    - content: bool
+      key: A
+    - content: float
+      key: B
+    - content: address
+      key: C
+    - content: char
+      key: D
+
+- uuid: 5413170f-f8c0-40fd-b17a-9481650eaa72
+  type: SingleChoice
+  content: What is the default value of int data type in solidity?
+  hint: noHint
+  explanation: null and undefined do not exist in solidity.
+  answerKeys:
+    - B
+  sub_topics:
+    - "value-types"
+  difficulty_level: Low
+  choices:
+    - content: "null"
+      key: A
+    - content: "0"
+      key: B
+    - content: undefined
+      key: C
+    - content: "1"
+      key: D
+
+- uuid: 165cc35e-7d95-4273-a61a-621916e5ef05
+  type: SingleChoice
+  content: Which of the following statements is incorrect regarding data types in solidity?
+  hint: noHint
+  explanation: The range of uintx in solidity is [0,2**x -1], the range of intx is [-2**(x-1),2**(x-1)-1] , boolean values only take true/false values and address literals take 20 bytes hexadecimal values.
+  answerKeys:
+    - C
+  sub_topics:
+    - "value-types"
+  difficulty_level: Medium
+  choices:
+    - content: The range of uint32 data type is from 0 upto 2**32 -1
+      key: A
+    - content: Bool data type can only take true and false values.
+      key: B
+    - content: The range of int256 data type is from -2**256+1 to 2**256-1
```

Comment:
The generated content shows this option as 
![image](https://user-images.githubusercontent.com/745748/178570610-789464ce-79a5-4607-9f39-bcfc1a36e96c.png)

So `-2**256+1 to 2**256-1` needs to be updated